release/release_after_kokoro.sh

```diff
@@ -89,7 +89,8 @@ ask_proceed
 ###############################################################################
 echo
 echo "#"
-echo "# Run Eclipse from the command line to generate the following files:"
+echo "# Run Eclipse from the command line to generate a p2 repository from"
+echo "# the signed artifacts:"
```

Comment:
Okay, then how about something like

"a p2 repository from the signed artifacts. The new repository will have the following files:"

or anything that can make clear that "$LOCAL_REPO/*" are being generated by this command.